File: empty.py
from time import perf_counter_ns

def my_function():
  for i in range(100_000_000):
    pass
		
# Start the stopwatch / counter
t1_start = perf_counter_ns()

my_function()

# Stop the stopwatch / counter
t1_stop = perf_counter_ns()

print((t1_stop-t1_start) / 1_000_000_000, 's')

File: Empty.java
import java.io.*;

public class Empty {
    // main function
    public static void main(String[] args)
    {
        // Start measuring execution time
        long startTime = System.nanoTime();

        count_function(100_000_000);

        // Stop measuring execution time
        long endTime = System.nanoTime();

        // Calculate the execution time in milliseconds
        long executionTime
            = (endTime - startTime);

	double seconds = (double)executionTime / 1_000_000_000.0;
        System.out.println(seconds + " s");
        
    }
  
    public static void count_function(long x)
    {
        for (long i = 0; i < x; i++)
            ;
    }
}
